### PR TITLE
(Refactor) Album Controller : Operations priority might differ from w…

### PR DESCRIPTION
**Pull request by VictoRD11**
(Refactor) Album Controller: Operations priority might differ from what you expect.

### CODE REVIEW

File: app/Http/Controllers/AlbumController.php [hunk 1]
```diff
@@ -135,7 +136,7 @@ public function destroy(Request $request, $id)
         $user = $request->user();
         $album = Album::findOrFail($id);
 
-        abort_unless($user->group->is_modo || $user->id === $album->user_id && Carbon::now()->lt($album->created_at->addDay()), 403);
+        abort_unless($user->group->is_modo || ($user->id === $album->user_id && Carbon::now()->lt($album->created_at->addDay())), Response::HTTP_FORBIDDEN);
```
Comment (HDVinnie) [inline app/Http/Controllers/AlbumController.php:139]: Why are we using `Response::HTTP_FORBIDDEN` here over `403`.

Comment (VictoRD11) [reply app/Http/Controllers/AlbumController.php:139]: I think it's easier to read the code, I can return it to 403.

Comment (HDVinnie) [reply app/Http/Controllers/AlbumController.php:139]: `403` is used commonly through out the codebase so it preferred. Unless going to refactor all usages.

Comment (VictoRD11) [reply app/Http/Controllers/AlbumController.php]: Ok, reverted to number.

**Pull request merged**